src/components/mortgage-investment-compare/input-card.js

```diff
@@ -129,12 +165,11 @@ const InputCard = props => {
                 <b>Mortgage Option 1</b>
               </small>
               <InputGroup className='my-2'>
-                <Input
-                  name='options.option1.interestRate'
+                <ValidatedNumberInput
+                  name='option1.interestRate'
```

Comment:
it's highly unlikely that a bank will pay you to take out a mortgage so I would pass in `mustBeGreaterThan={0}` and `min={0}` here as well

Reply:
🤷‍♂ who knows, you might be danish: https://www.theguardian.com/money/2019/aug/13/danish-bank-launches-worlds-first-negative-interest-rate-mortgage

Reply:
(I checked though that our calculator works for negative interest rates)

Reply:
woah thanks for the article

Reply:
I'm not sure if our calc works well for negative interest rates. I think we might need to make the budget negative when that happens and I believe I'm using `Math.abs()` because I didn't think we could get "positive payments". Let me know if I'm missing something in this logic but, if we made the APR negative, shouldn't the bank be paying you? so the payment would be negative in the summary? That's not what the tool does right now

Reply:
ah wait no, the bank shouldn't be paying you with a negative APR right? because it's still a loan that you have to pay back. It's just that each payment will receive a discount of whatever the negative interest rate is